### PR TITLE
Vea: delete

### PR DESCRIPTION
**Pull request by CloCkWeRX**

### CODE REVIEW

File: locations/spiders/duluth_trading.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class DuluthTradingSpider(CrawlSpider):
     rules = [Rule(LinkExtractor(allow="/locations/"), callback="parse", follow=True)]
 
     def parse(self, response):
+
```
Comment (CloCkWeRX) [inline locations/spiders/duluth_trading.py:20]:
```suggestion
```

File: locations/spiders/vea.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 class VeaSpider(scrapy.Spider):
     name = "vea"
     item_attributes = {"brand": "Vea Cencosud"}
-    allowed_domains = ["www.supermercadosvea.com"]
+    allowed_domains = ["www.supermercadosvea.com.ar"]
     start_urls = ("http://www.supermercadosvea.com.ar/sucursales-obtener.html",)
     requires_proxy = True
```
Comment (CloCkWeRX) [inline locations/spiders/vea.py:11]:
```suggestion
    requires_proxy = "AR"
```

**Comment by CloCkWeRX** [comment]
Well, this one is very tricky unless you have a proxy in Argentina to figure out.

**Comment by iandees** [comment]
Their website works for me from home, but I don't see any store location listings on their page. I think we could probably delete this unless someone is more local?

**Pull request merged**